mantidproject/mantid
# Update the ISIS_Powder system test reference files (Not including Pearl)

Ensured that the system tests for ISIS_Powder actually test for comparison.
Additionally I have remove the `_compare_ws` function from each of the system tests in favour of using the standard Mantid validate (which calls `CompareWorkspace` anyway)

**To test:**

* Ensure the system tests for ISIS_Powder run on the build server
* Ensure that the reference files have been updated
* Ensure that the ISIS_Powder system tests run locally

Fixes #21229

**Release Notes** 
*Does not need to be in the release notes - internal change.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
